external-crates/move/move-compiler/tests/move_2024/typing/const_in_const_circular.exp

```diff
@@ -0,0 +1,30 @@
+error[E08001]: cannot compute constant value
+  ┌─ tests/move_2024/typing/const_in_const_circular.move:3:11
+  │
+3 │     const C0: u64 = C1 + 1;
+  │           ^^ Constant involved in constant cycle
```

Comment:
Is it possible to print out at least one edge in the cycle? Just helpful to at least point to somewhere else in the cycle (maybe where it is used?)

Reply:
I reworked these error messages.